### PR TITLE
Matched "crisp-edges" and "pixelated" image-rendering to W3C spec

### LINKED ISSUE #24734: Description of image-rendering algorithms do not follow the W3C specs
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/CSS/image-rendering

### What specific section or headline is this issue about?

Values

### What information was incorrect, unhelpful, or incomplete?

The values crisp-edges and pixelated are too specific.
It says that crisp-edges uses nearest neighbour scaling, but the W3C spec reads:

> The image must be scaled with an algorithm that preserves contrast and edges in the image, and which does not smooth colors or introduce blur to the image in the process. This is intended for images such as pixel art or line drawings. 

For pixelated, there's a description of how two steps are used in the rendering, first nearest neighbour to the nearest greater integer multiple scale, then a smooth render pass to render down to the actual intended scale. However, the W3C spec reads:

> The image must be scaled with the "nearest neighbor" or similar algorithm, to preserve a "pixelated" look as the image changes in size.

I.e. nearest neighbour rendering is only mentioned for pixelated rendering, but even there, it's not a required algorithm, nor is there any mention of what steps are to be taken for non-integer multiples (and nearest neighbour scaling works best for powers of two, not any arbitrary choice of integer values).

### What did you expect to see?

I was expecting to see the W3C spec descriptions, or a variation describing the same results. The current description is too specific and does not describe actual browser implementations. Otherwise crisp-edges and pixelated should look the same across rendering engines, but crisp edges look much worse in Gecko than Blink.

### Do you have any supporting links, references, or citations?

https://www.w3.org/TR/css-images-3/#valdef-image-rendering-crisp-edges
https://www.w3.org/TR/css-images-3/#valdef-image-rendering-pixelated

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/css/image-rendering`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/image-rendering
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/css/image-rendering/index.md
* Last commit: https://github.com/mdn/content/commit/73091fbe590d96857d743eaeec5aee4a8101994f
* Document last modified: 2023-02-21T05:50:15.000Z

</details>

### PR DESCRIPTION
**Pull request by Sardtok**
Fixes #24734 by simply using the W3C specification's description of how browsers should render images with these values.

### Description

Changes the description of rendering algorithms used for `image-rendering` of `pixelated` or `crisp-edges` to match the W3C specification.

### Motivation

The old description used specific algorithms, but rendering engines use different algorithms, at least for `crisp-edges`, which the W3C spec is open to.

### Additional details

https://www.w3.org/TR/css-images-3/#valdef-image-rendering-crisp-edges
https://www.w3.org/TR/css-images-3/#valdef-image-rendering-pixelated

### Related issues and pull requests

Fixes #24734

### CODE REVIEW

File: files/en-us/web/css/image-rendering/index.md [hunk 1]
```diff
@@ -38,9 +38,9 @@ image-rendering: unset;
 - `high-quality` {{Experimental_Inline}}
   - : Identical to `smooth`, but with a preference for higher-quality scaling. If system resources are constrained, images with `high-quality` should be prioritized over those with any other value, when considering which images to degrade the quality of and to what degree.
 - `crisp-edges`
-  - : The image is scaled with the nearest-neighbor algorithm.
+  - : The image must be scaled with an algorithm that preserves contrast and edges in the image, and which does not smooth colors or introduce blur to the image in the process. This is intended for images such as pixel art or line drawings.
 - `pixelated`
-  - : Using the nearest-neighbor algorithm, the image is scaled up to the next integer multiple that is greater than or equal to its original size, then scaled down to the target size, as for `smooth`. When scaling up to integer multiples of the original size, this will have the same effect as `crisp-edges`.
+  - : The image must be scaled with the "nearest neighbor" or similar algorithm, to preserve a "pixelated" look as the image changes in size.
```
Comment (estelle) [inline files/en-us/web/css/image-rendering/index.md:43]:
```suggestion
  - : The image is scaled with the "nearest neighbor" or similar algorithm, preserving a "pixelated" look as the image changes in size.
```

**Review by estelle — CHANGES_REQUESTED** [review]
Thank you for your contribution.

The specification text is written for browser developers, with some "non-normative" (meaning "informative") content aimed at web authors. The MDN text is geared toward developers encountering concepts for the first time. I made some suggested edits with that in mind.

**Comment by Sardtok** [comment]
Thanks, @estelle. The descriptive wording is definitely more suitable for documentation on usage.

Added your changes and rebased the branch.

**Review by estelle — APPROVED** [review]
Thank you. I am so sorry that this PR fell thru the cracks. I think I accidentally muted it. Thank you for your contribution.

**Pull request merged**